.github/workflows/multi-arch-test-build.yml

```diff
@@ -0,0 +1,56 @@
+name: Test Build
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: ${{ matrix.arch }} build
+    runs-on: ubuntu-latest
```

Comment:
This maps to Ubuntu 18.04. Would it make sense to switch to 16.04? I believe 16.04 is still supported.

macos-10.15 is also an option. Actually that would be really interesting as I've seen several patches recently that fix compilation under OS X.

macos would be interesting should we decide to keep CircleCI, which I believe runs on Debian.

Reply:
This is something else. This pull requests uses our SDKs running within a Debian 10. The Ubuntu is just the host OS starting the Docker container.

It is additionally possible to test OS X, however at this point we do not offer any SDKs for that. I don't think it is feasible to rebuild the entire toolchain for every pull requests, what do you think?

Reply:
Ah never mind then.